packages/mobile-app-did/js/pages/Chat/components/SendRedPacketGroupSection/index.tsx

```diff
@@ -53,10 +56,13 @@ const AMOUNT_LABEL_MAP = {
 };
 
 export default function SendRedPacketGroupSection(props: SendRedPacketGroupSectionPropsType) {
-  const { type, groupMemberCount, onPressButton } = props;
+  const { type, groupMemberCount, isCryptoGift, onPressButton } = props;
+  console.log('wfs=== isCryptoGift', isCryptoGift);
   const { getTokenInfo } = useGetRedPackageConfig();
   const [tokenPriceObject] = useGetCurrentAccountTokenPrice();
 
+  const isNewUserOnly = useRef<boolean>(false);
```

Comment:
is useRef better than useState here?

Reply:
`isNewUserOnly` state need not keep. it's rendering  irrelevant